.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: CI
+
+on: push
+
+jobs:
+  linter:
+    runs-on: ubuntu-latest
+    container: uniplaces/node:10.3-chromium-alpine-3
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+
+      - name: Running linter
+        run: |
+          npm ci
```

Comment:
We can use cache for the `npm ci`. Just a suggestion

Reply:
I added it. However, I realized the cache step breaks if used along with a container like `uniplaces/node-10.3.0`, at least when both jobs are using it. Therefore I added a specific step to set up de node version.